sample-app/src/App.vue

```diff
@@ -29,6 +29,8 @@ import main from "./main.js";
 import DatePlugin from "@sap-devx/inquirer-gui-date-plugin";
 import LabelPlugin from "@sap-devx/inquirer-gui-label-plugin";
 import FileBrowserPlugin from "@sap-devx/inquirer-gui-file-browser-plugin";
+//import RadioGroup from "@sap-devx/inquirer-gui-radio-button-plugin";
+import RadioGroup from "../../radio-button-plugin/src/packages/RadioGroup";
```

Comment:
seems like `import` from the commented line should be used here

Reply:
changed